decidim-core/lib/decidim/core.rb

```diff
@@ -113,7 +114,16 @@ def self.seed!
   # 6589: ID of the resource
   config_accessor :resource_reference_generator do
```

Comment:
Maybe we should migrate to `reference_generator`, as we're no longer generating references only for *resources*?

(Aliasing it with a deprecation notice)

Reply:
Maybe by adding at this at the block:

`ActiveSupport::Deprecation.warn "the warning..."`

And calling reference_generator within?

Reply:
But if you overwrite the `config_accessor`, then the deprecation warning will not be called...

Reply:
Also, I feel like a participatory space is still a resource. Maybe not a feature resource, but a resource nonetheless